zerver/lib/api_test_helpers.py

```diff
@@ -515,6 +523,19 @@ def add_reaction(client, message_id):
 
     assert result['result'] == 'success'
 
+def remove_reaction(client, message_id):
+    # type: (Client, int) -> None
+    request = {
+        'message_id': message_id,
+        'emoji_name': 'joy',
+        'emoji_code': '1f602',
+        'reaction_type': 'unicode_emoji'
+    }
+
+    result = client.remove_reaction(request)
+
+    assert result['result'] == 'success'
+
```

Comment:
This function needs to be called near the bottom of this file. Currently, this isn't even being run.